src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -393,16 +410,8 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
     }
 
     public currentTab(tabId:string|undefined):string {
-        // if we have no tab defined (query submission, no tab click)
-        // we need to evaluate which should be the default tab
-        // this can only be determined by know the count of physical studies in the query
-        // (for virtual studies we need to fetch data determine constituent physical studies)
         if (tabId === undefined) {
-            if (this.resultsViewPageStore.studies.result!.length > 1 && this.resultsViewPageStore.hugoGeneSymbols.length === 1) {
-                return ResultsViewTab.CANCER_TYPES_SUMMARY; // cancer type study
-            } else {
-                return ResultsViewTab.ONCOPRINT; // this will resolve to first tab
-            }
+            return ResultsViewTab.ONCOPRINT;
```

Comment:
feels like we should just do this inside store.tabId

Reply:
agreed